xml/cha_images.xml

```diff
@@ -136,28 +145,35 @@
     <varlistentry>
      <term><literal>VIRTUALIZATION</literal> (&awsa;-only)</term>
      <listitem>
-      <para>
-       The virtualization type is either <literal>pv</literal>
-       (para-virtualized) or <literal>hvm</literal> (hardware-assisted virtual
-       machine).
+       <para>
+        &suse; no longer supports or publishes para-virtualized images. The 
+        virtualization type was encoded as <literal>pv</literal>
+        (para-virtualized) or <literal>hvm</literal> (hardware-assisted virtual
+        machine). The <literal>hvm</literal> part of the image name has been
+        retained in an effort to not break backward compatibility.
       </para>
      </listitem>
     </varlistentry>
     <varlistentry>
      <term><literal>STORAGE</literal> (&awsa;-only)</term>
      <listitem>
-      <para>
-       The storage type is either <literal>ssd</literal> for SSD-backed
-       or <literal>mag</literal> for magnetic-backed images.
+       <para>
+         &suse; no longer publishes images that are based on hard disk
+         (magnetic) backed storage. This used to be encoded as
+         <literal>mag</literal>. All published images are backed by SSD.
+         The <literal>ssd</literal>  part of the image name has been
+         retained in an effort to not break backward compatibility.
       </para>
      </listitem>
     </varlistentry>
     <varlistentry>
-     <term><literal>ARCHITECTURE</literal> (&awsa;-only)</term>
+     <term><literal>ARCHITECTURE</literal></term>
      <listitem>
       <para>
-       Either <literal>i386</literal>,
-       <literal>x86_64</literal>, or <literal>arm64</literal>.
+        Either <literal>x86_64</literal> or <literal>arm64</literal>.
+        &suse; no longer supports or publishes 32 bit <literal>x86</literal>
+        images. Images with the <literal>i386</literal> identifier are visible
+        in &pint; data.
```

Comment:
I think we should align this text with the others.

Is the last sentence really relevant any longer? All of the i386 images are deleted and not available to create new instances. They were ancient (the last ones from 2015) and deleted two years ago. I suggest we remove that sentence. OK?

Reply:
I generally have no problem removing the `i386` mention here, but we should reference it somewhere as those images are still in the pint data. A user of pint listing everything will find `i386`. It should be obvious what it means but we probably should not make that assumption.